script.js

```diff
@@ -0,0 +1,86 @@
+// JavaScript for dynamically featured speakers
+const speakers = [
+  {
+    name: 'Anjali Sastry',
+    title: 'Senior Lecturer at MIT Sloan School of Management',
+    description: 'Expert in global health delivery, applying management and systems thinking to improve public health.',
+    image: 'https://via.placeholder.com/100',
+  },
+  {
+    name: 'Neal Khosla',
+    title: 'CEO of Curai',
+    description: 'Focused on using AI to improve healthcare access and outcomes at Curai.',
+    image: 'https://via.placeholder.com/100',
+  },
+  {
+    name: 'Dana Teppert',
+    title: 'Chief Strategy Officer at Global Innovation Fund',
+    description: 'Leads global innovation efforts to bring sustainable and impactful solutions to developing countries.',
+    image: 'https://via.placeholder.com/100',
+  },
+  {
+    name: 'Alex Amouyel',
+    title: 'Executive Director of Solve at MIT',
+    description: 'Driving innovation challenges to solve world problems through open innovation.',
+    image: 'https://via.placeholder.com/100',
+  },
+  {
+    name: 'Matthew Kalil',
+    title: 'Director of Innovation at UNDP',
+    description: 'Focused on creating systems and platforms for sustainable development and collaboration.',
+    image: 'https://via.placeholder.com/100',
+  },
+  {
+    name: 'Raj Panjabi',
+    title: 'CEO of Last Mile Health',
+    description: 'Global health innovator, improving access to health services in remote areas.',
+    image: 'https://via.placeholder.com/100',
+  },
+];
```

Comment:
@Aqsamaan12 this  is nice you are using content dynamically, but that content should be render to your main page